straxen/plugins/event_processing.py

```diff
@@ -703,6 +713,22 @@ def infer_dtype(self):
                       (f'{peak_type}cs2', np.float32, f'Corrected area of {peak_name} S2 [PE]'), ]
         return dtype
 
+    @staticmethod
+    def rotate(x_arr, y_arr, theta):
+        new_x = np.cos(theta)*x_arr+np.sin(theta)*y_arr
+        new_y = -np.sin(theta)*x_arr+np.cos(theta)*y_arr
+        return np.array([new_x, new_y])
```

Comment:
Did you have a look at the function I linked?

Reply:
I will update it to use the common one.

Reply:
Done